tempesta_fw/sock.c

```diff
@@ -196,6 +188,12 @@ ss_do_send(struct sock *sk, SsSkbList *skb_list)
 	       smp_processor_id(), __func__,
 	       sk, tcp_send_head(sk), sk->sk_state);
 
+	/* If connection close flag is specified, then @ss_do_close is used to
+	 * set FIN on final SKB and push all pending frames to the stack. */
```

Comment:
Please use 

        /*
         * blah blah blah
         */

- style for multi line comments